src/Raven.Server/Documents/Handlers/PullReplicationHandler.cs

```diff
@@ -166,9 +183,24 @@ public Task GeneratePullReplicationCertificate()
                 throw new BadRequestException("This endpoint requires secured server.");
 
             ServerStore.LicenseManager.AssertCanAddPullReplicationAsHub();
+            
+            var validMonths = GetIntValueQueryString("validMonths", required: false) ?? 0;
```

Comment:
remove '?? 0' from both of those and use .HasValue for validation purposes

Reply:
done